build.jdk8u172.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
@pedroneil This probably wouldn't work for everyone.  You might want to pull this back for now.

Reply:
I'll remove build.jdk8u172.cmd